sites/tvtv.us/tvtv.us.config.js

```diff
@@ -8,17 +8,33 @@ module.exports = {
   delay: 1500, // 1500 ms (otherwise the server returns error 429: https://github.com/iptv-org/epg/issues/2176)
   days: 2,
   url: function ({ date, channel }) {
+    if (!dayjs.isDayjs(date)) {
+      throw new Error('Invalid date object passed to url function')
+    }
+
     return `https://www.tvtv.us/api/v1/lineup/USA-NY71652-X/grid/${date.toJSON()}/${date
-      .add(1, 'd')
+      .add(1, 'day')
       .toJSON()}/${channel.site_id}`
   },
+  request: {
+    method: 'GET',
+    headers: {
+      Accept: '*/*',

```

Comment:
Hi, I am not sure whether this helps or not, and whether I am even supposed to contribute here. Apologies if not. I had to remove this line in order to make it work, not sure why.